src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
```

Comment:
No. The largest still didn't provide enough room for the close X. I actually find our modular sizes to not be a wide enough range - xs is too big for a lot of things and ls is too small. Or maybe I'm not using them correctly.